packages/python-pixelmatch/PKGBUILD

```diff
@@ -0,0 +1,28 @@
+# This file is part of BlackArch Linux ( https://www.blackarch.org/ ).
+# See COPYING for license details.
+
+pkgname=python-pixelmatch
+_pkgname=${pkgname#python-}
+pkgver=0.3.0
+pkgrel=1
+pkgdesc='A fast pixel-level image comparison library, originally created to compare screenshots in tests.'
+arch=('any')
+url='https://github.com/whtsky/pixelmatch-py'
+license=('ISC')
+depends=('python')
+makedepends=('python-poetry-core' 'python-setuptools')
+options=(!emptydirs)
+source=("https://files.pythonhosted.org/packages/source/${_pkgname::1}/$_pkgname/$_pkgname-$pkgver.tar.gz")
+sha512sums=('8d5f9ba9dc727900292edb8ad0787c1ba1e5b50099275265fa40979cf571d441f5fd70995214049e5fa3eb5ce6680c58e889c6fb650922b8420c01dc630c8354')
+
+build() {
+  cd "$_pkgname-$pkgver"
+
+  python setup.py build
+}
+
+package() {
+  cd "$_pkgname-$pkgver"
+
+  python setup.py install --root="$pkgdir" --prefix=/usr -O1 --skip-build
+}
```

Comment:
Could you add an ending newline for every PKGBUILD please?

Reply:
> Could you add an ending newline for every PKGBUILD please?

If I click on Edit File on them, I see already an ending newline. Is there a particular way to add it?

Reply:
Github editor always shows an extra one that is never included.

Reply:
Added